container-host-files/etc/hcf/config/opinions.yml

```diff
@@ -269,6 +268,7 @@ properties:
     url: https://this.is.unused
   metron_agent:
     deployment: hcf-deployment
+    health_port: 9999
```

Comment:
https://github.com/cloudfoundry/loggregator/commit/435af2536c7823448d6da5db96cac43a33295c85

I think this should be changed to `14824`, the default that upstream eventually agreed upon. The validation checks should flag this as unchanged from the default once we start using a release with this commit.